app/assets/javascripts/discourse/models/post.js.es6

```diff
@@ -333,6 +333,10 @@ Post.reopenClass({
     return Discourse.ajax("/posts/" + postId + "/revisions/" + version + "/show", { type: 'PUT' });
   },
 
+  revertToRevision(postId, version) {
+    return Discourse.ajax("/posts/" + postId + "/revisions/" + version + "/revert", { type: 'PUT' });
```

Comment:
String interpolation might be useful here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Expression_interpolation